MICADO_Sci/MICADO_Sci_SPEC.yaml

```diff
@@ -50,6 +50,9 @@ effects:
     kwargs :
         filename : "TRACE_SCI_3arcsec.fits"
         center_on_wave_mid: True
+        x_colname : "x0"
+        y_colname : "y0"
+        s_colname : "s0"
```

Comment:
This is not a proper fix, instead the trace files themselves should be updated to the new format.

Reply:
Created #48 for updating these trace files, so this can be merged.